TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/RunningActivity.java

```diff
@@ -364,6 +387,40 @@ public void onClick(View view) {
         }
     }
 
+    public void onClickOkLoadOccupancyGridDialog(final String occupancyGridName) {
+        if (occupancyGridName == null || occupancyGridName.isEmpty()) {
+            Log.e(TAG, "Name is null or empty, unable to load the occupancy grid");
+            displayToastMessage(R.string.name_error);
```

Comment:
Instead of showing a toast message which can easily be missed, I would display the dialog again. Possibly show extra text in red that previous map could not be loaded. Cancel button is still there if a user would like to cancel the operation.

Alternative: just check fail here or raise an exception because it should never happen.